### PR TITLE
Adding support to deploy ZGC nodes using KVM VMs in lab environment\n…

### LINKED ISSUE #47: Zeta integration:  add support for deploying ZGC nodes as KVM VMs
New script tool is added to automate multi-VM deployment with cloudinit
usage: 
./tools/kvm_deploy.sh <num_of_zgc_nodes>

### PR DESCRIPTION
**Pull request by liangbin-pub**
…Closes #47

### CODE REVIEW

**Review by zzxgzgz — APPROVED** [review]
looks good to me

File: tools/kvm_deploy.sh [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+checkdep virt-install virtinst
+checkdep qemu-img qemu-kvm
+checkdep virsh libvirt-bin
+checkdep wget wget
+
+if [ -n "$MISSING" ]; then
+    echo
+    [ -n "${FOUND}" ] && echo "found: ${FOUND}"
+    echo "install missing deps with:"
+    echo "  apt-get update && apt-get install ${MISSING}"
```
Comment (phudtran) [inline tools/kvm_deploy.sh]: Do we want to add -y here to the apt-gets to skip user confirmation?

File: tools/kvm_deploy.sh [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+version: 2
+ethernets:
+    ens2:
+        addresses:
+          - 10.213.43.${ip}/24
+        gateway4: 10.213.43.1
+        nameservers:
+          addresses:
+            - 10.208.115.34
+            - 10.208.115.35
```
Comment (phudtran) [inline tools/kvm_deploy.sh:112]: Are these specific to our current lab setup? 
Might be good to define them as a constant.

**Review by phudtran — APPROVED** [review]
LGTM

File: tools/kvm_deploy.sh (hunk 1 above)
Comment (liangbin-pub) [reply tools/kvm_deploy.sh]: agree

File: tools/kvm_deploy.sh (hunk 2 above)
Comment (liangbin-pub) [reply tools/kvm_deploy.sh:112]: yes and agree, best to add a template, will take care of it in later change

**Pull request merged**